src/connector/src/parser/unified/util.rs

```diff
@@ -86,16 +87,27 @@ pub fn apply_row_operation_on_stream_chunk_writer(
 pub fn apply_row_accessor_on_stream_chunk_writer(
     accessor: impl Access,
     writer: &mut SourceStreamChunkRowWriter<'_>,
-) -> std::result::Result<WriteGuard, RwError> {
+) -> Result<WriteGuard, RwError> {
     writer.insert(|column| {
-        let res = match accessor.access(&[&column.name], Some(&column.data_type)) {
+        let res: Result<Datum, RwError> = match accessor
+            .access(&[&column.name], Some(&column.data_type))
+        {
             Ok(o) => Ok(o),
             Err(AccessError::Undefined { name, .. }) if !column.is_pk && name == column.name => {
                 // Fill in null value for non-pk column
                 // TODO: figure out a way to fill in not-null default value if user specifies one
                 Ok(None)
```

Comment:
Seems this arm is redundant.

Reply:
I would like to keep this because maybe we need an option to control whether we need to drop the whole row.